cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -100,6 +106,47 @@ def optimization_at(
         )
 
 
+@cirq.transformer
+def strategy_executor(
+    circuit: 'cirq.Circuit',
+    *,
+    execution_strategy: ExecutionStrategy = None,
+    context: Optional['cirq.TransformerContext'] = None,
+) -> None:
+    """Executes an acquaintance strategy"""
+
+    if execution_strategy is None:
+        raise ArgumentError(execution_strategy, 'Execution strategy cannot be None')
+
+    mapping = execution_strategy.initial_mapping.copy()
+
+    def map_func(op: 'cirq.Operation', index) -> 'cirq.OP_TREE':
+        if isinstance(op.gate, AcquaintanceOpportunityGate):
+            logical_indices = tuple(mapping[q] for q in op.qubits)
+            logical_operations = execution_strategy.get_operations(logical_indices, op.qubits)
+            clear_span = int(not execution_strategy.keep_acquaintance)
+            new_operation = logical_operations.on(*logical_indices)
```

Comment:
`execution_strategy.get_operations` returns a `cirq.OP_TREE` and therefore `logical_operations` is `cirq.OP_TREE`; which cannot directly be applied on `logical_indices`. 

I think we should directly return `logical_operations` instead of `new_operation` here?